ArduCopter/mode.h

```diff
@@ -886,19 +895,21 @@ class ModeGuided : public Mode {
 
     // enum for GUID_OPTIONS parameter
     enum class Options : int32_t {
-        AllowArmingFromTX = (1U << 0),
+        AllowArmingFromTX   = (1U << 0),
         // this bit is still available, pilot yaw was mapped to bit 2 for symmetry with auto
-        IgnorePilotYaw    = (1U << 2),
+        IgnorePilotYaw      = (1U << 2),
         SetAttitudeTarget_ThrustAsThrust = (1U << 3),
+        DoNotStabilizePositionXY = (1U << 4),
```

Comment:
I think this means we want users to be able to turn off the position and/or velocity control in Guided mode.  If "yes" then we should modify the GUID_OPTIONS parameter description to make the checkboxes visible to users.